snap/snapcraft.yaml

```diff
@@ -58,25 +65,34 @@ parts:
       fi
 
       /usr/bin/dotnet publish src/Dotnet.Installer.Console --output "${SNAPCRAFT_PART_INSTALL}" \
-        --configuration Release -r "${RUNTIME_RID}" -p:DebugSymbols=false -p:DebugType=none
+        --configuration Debug -r "${RUNTIME_RID}" -p:DebugSymbols=true -p:DebugType=full
```

Comment:
question: Why did you change `--configuration Release` to `--configuration Debug`?

Reply:
Ugh, I committed this by accident 🤦🏼 I was debugging something and left this in there. Thanks for catching it!